ivy_tests/test_ivy/test_frontends/test_numpy/test_indexing_routines/test_inserting_data_into_arrays.py

```diff
@@ -16,7 +16,7 @@
         min_dim_size=2,
         max_num_dims=2,
     ),
-    val=helpers.floats(),
+    val=helpers.floats(min_value=-10, max_value=10),
```

Comment:
We should not restrict it to such a small number,
We only restrict data generations in hypothesis for those functions which are numerically unstable on big numbers
This is just the value to be populated in the diagonal, try not restricting it and see if it passes the test.

Reply:
Most of the time test are passed without restriction but occasionally some backends are unable to handle floating point precision. In my opinion, I think it's necessary to restrict floating point numbers. But I'm more than happy to know if there's any other way to handle this thing.